content/attacks/2023-11-18-Kronos.md

```diff
@@ -27,11 +27,14 @@ At the time of the attack, Kronos Research lost around $26 million. According to
 
 ## Timeline
 
-   - **November 18, 2023, 05:05 PM UTC:** Initial funds transfer [begins.](0x6722a7a77580234221053eafecda7aebecaf8d002223d261e2b25a96cc7f35be)
+   - **November 18, 2023, 05:05 PM UTC:** Initial funds transfer [begins.](https://etherscan.io/address/0x2b0502fdab4e221dcd492c058255d2073d50a3ae) 
```

Comment:
It's not a link to a transaction, the link contains `/address/` . It should be `/tx/` instead, and transaction hash included, which is little longer than address. Example: `https://etherscan.io/tx/0x6722a7a77580234221053eafecda7aebecaf8d002223d261e2b25a96cc7f35be` . It's just an example an is not funds transfer transaction.